tests/TestBase.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python2
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+import sys
+import os.path
+import re
+parent = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, parent)
+import requests
+
+'''
+Created on 10 Jul 2017
+
+@author: adrian
```

Comment:
We don't normally include headers like these - that's what git is for

Reply:
I'll remove them and update my Eclipse template for this project.